ios/Firestack/FirestackRemoteConfig.m

```diff
@@ -0,0 +1,232 @@
+//
+//  FirestackRemoteConfig.m
+//  Firestack
```

Comment:
I managed to put some objective-c code together using example, but it is throwing exceptions.